src/components/Table/GridTable.tsx

```diff
@@ -300,46 +283,23 @@ export function GridTable<R extends Kinded, S = {}, X extends Only<GridTableXss,
     setRowCount,
     observeRows,
     persistCollapse,
-    api: callerApi,
     resizeTarget,
     activeRowId,
   } = props;
 
-  // Create a ref that always contains the latest rows, for our effectively-singleton RowState to use
-  const rowsRef = useRef(rows);
-  rowsRef.current = rows;
-  const [rowState] = useState(() => new RowState(rowsRef, persistCollapse, activeRowId));
```

Comment:
The `RowState` is now managed as an internal impl detail of `GridTableApi`, so that when the caller creates an api immediately with `const api = useGridTableApi()`, it will already have a `useComputed`-able `RowState` living inside it.